iojs-el:Add new translations for template.json

AntouanK opened the pull request.

AntouanK committed:
iojs-el:Add new translations for template.json

fhemberger commented:
@nodejs/iojs-el, please review.

AvraamMavridis commented on content/el/template.json:
```diff
@@ -1,15 +1,55 @@
-  "browser-title":"io.js - JavaScript I/O",
-  "logo-text":"io.js",
+  "browser-title": "io.js - JavaScript I/O",
+  "contribute-message": "Βλέπεις κάτι που σου αρέσει? Θέλεις να βοηθήσεις? Επισκέψου την https://github.com/nodejs/website για να συνεισφέρεις.",
```
Γιατί όχι ελληνικό ερωτηματικό;  `?` -> `;`

fhemberger commented:
Can you please remove the old `index.md` file if the new template works for you? Thanks!

AntouanK committed:
iojs-el:Switch to greek question mark symbol.

AntouanK commented:
@sirodoht Added a new commit for that. Thanks.

AntouanK commented:
@fhemberger If I remove the `index.md` file, do I have to add anything else? Or does the `template.json` cover all the translations?

fhemberger commented:
`template.json` replaces the `index.md` entirely.

AntouanK committed:
iojs-el:Remove index.md, add few more translations.

AntouanK commented:
@fhemberger I removed `index.md` and added another missing translation ( was in `index.md` )
Only thing missing is the "Critical update" message which I think is not in the JSON file.

fhemberger commented:
No, the update message comes from a different source. For critical security updates, we don't have the time to go through the translation process, so this message is English for all language variants.

AntouanK commented:
@fhemberger Are we ok with that PR? Do we need something more?

fhemberger commented:
@AntouanK Just waiting for a +1 from @nodejs/iojs-el …

AvraamMavridis commented:
:+1:

fhemberger commented:
Aaaand there it is, merging!

AntouanK commented:
@AvraamMavridis Thanks
